includes/abstracts/abstract-wc-order.php

```diff
@@ -780,14 +780,38 @@ public function get_item_count( $item_type = '' ) {
 	}
 
 	/**
-	 * Get an order item object, based on it's type.
+	 * Get an order item object, based on it's ID. The item must belong to the current
+	 * order. If the item cannot be found or it doens't belong to current order
```

Comment:
How 'breaking' is this - we need to determine if this belongs in a minor or major release or if we should keep the old handling around for now..

@thenbrent thoughts?